test/load/loadtests.py

```diff
@@ -268,3 +309,15 @@ def test_password_change_flow(self):
     def test_support_doc_flow(self):
         base_url = self.server_url[:-3]
         self.session.get(base_url + "/.well-known/browserid")
+
+    def _get_code_from_email(self, emailAcct, header):
+        # Due to possible race conditions, and previous test runs
+        # leaving junk in the restmail account, we look for the last
+        # matching email rather than the first.  There's still a small
+        # chance that we pull out the wrong code here.
+        for _ in xrange(10):
```

Comment:
it's just a retry loop to wait for the email to arrive; similar logic to [1] except we want to most recent email, and we want to eventually fail out and I didn't want to sleep.

[1] https://github.com/mozilla/PyFxA/blob/master/fxa/tests/utils.py#L97